src/components/pages/Home.js

```diff
@@ -0,0 +1,59 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import fetchRepos from '../../redux/repositories/repositoriesThunk';
+import { filterRepos } from '../../redux/repositories/repositoriesSlice';
+
+const Home = () => {
+  let repos = useSelector((state) => state.repos.repos);
+  const filteredRepos = useSelector((state) => state.repos.filteredRepos);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRepos());
+  }, [dispatch]);
+  if (filteredRepos.length > 0) {
+    repos = filteredRepos;
+  }
+  return (
+    <>
+      <input
+        className="filter-input"
+        type="text"
+        placeholder="filter by fork count..."
+        onChange={(e) => {
+          dispatch(filterRepos(e.target.value));
+        }}
+      />
+
+      <ul className="repos-list">
+        {repos && repos.map(({
+          id, owner, stars, forks,
+        }) => (
+          <li key={id} className="repo-card">
+            <Link
+              to={`/repo_details/${owner}`}
+            >
+              <h1>
+                {owner}
+                {' '}
+                Fork
+              </h1>
+              <p>
+                Stars Count:
+                {stars}
+              </p>
+              <p>
+                Fork Count:
+                {forks}
+              </p>
+            </Link>
+
```

Comment:
[REQUIRED] Your app should follow the template in terms of:

- Typography: font face, size, and weight

-  Layout: composition and space between elements.
Kindly ensure that you have a top section on the homepage and cards arranged in two items per row along with the texts in each card aligned to the bottom right. Also, ensure that there are no spaces around each card.

![image](https://github.com/mohamedSabry0/gh-metrics/assets/62938735/9f588f3f-905b-41c4-9361-54cc9eef7ba3)